src_c/rect.c

```diff
@@ -956,25 +957,51 @@ pg_rect_clipline(pgRectObject *self, PyObject *args)
     return Py_BuildValue("((ii)(ii))", x1, y1, x2, y2);
 }
 
-static PyObject *
-pg_rect_contains(pgRectObject *self, PyObject *args)
+static int
+_pg_rect_contains(pgRectObject *self, PyObject *argrect, bool rev)
```

Comment:
Why is the `rev` required here? Actually we don't need this reversal, IMHO `big_rect.contains(arg_rect)` should be identical to `arg_rect in big_rect`. That way no reversal is needed, also it makes more logical sense

Reply:
Ah yep that does make more sense, and then that means no reversal is needed. I was going off the assumption that the 1st rect would be the the one containing the second but actually the second argument should be the one that contains the 1st to make logical sense. Fixed!